cpp/unreal_plugins/UrdfBot/Source/UrdfBot.Build.cs

```diff
@@ -0,0 +1,49 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+using UnrealBuildTool;
+using System;
+using System.Diagnostics;
+using System.IO;
+
+public class UrdfBot : ModuleRules
+{
+    public UrdfBot(ReadOnlyTargetRules Target) : base(Target)
+    {
+        PCHUsage = ModuleRules.PCHUsageMode.UseExplicitOrSharedPCHs;
+        bEnableExceptions = true;
+
+        PublicDependencyModuleNames.AddRange(new string[] { "Core", "CoreUObject", "CoreUtils", "Engine", "InputCore", "XmlParser" });
+        PrivateDependencyModuleNames.AddRange(new string[] { "Slate" });
+
+        //
+        // Eigen
+        //
+
+        PublicIncludePaths.Add(Path.Combine(ModuleDirectory, "..", "ThirdParty", "libeigen"));
+
+        //
+        // rbdl
+        //
+
+        PublicIncludePaths.Add(Path.Combine(ModuleDirectory, "..", "ThirdParty", "rbdl", "include"));
+        PublicIncludePaths.Add(Path.Combine(ModuleDirectory, "..", "ThirdParty", "rbdl", "build", "include"));
+        if (Target.Platform == UnrealTargetPlatform.Win64)
```

Comment:
mimic formatting style in OpenBot.Build.cs

Reply:
fixed